sandy/core/cov.py

```diff
@@ -1885,7 +1881,7 @@ def sparse_tables_dot(a, b, rows=1000):
     return dot_product
 
 
-def sparse_tables_dot_multiple(matrix_list, rows=1000):
+#def sparse_tables_dot_multiple(matrix_list, rows=1000):

```

Comment:
@EnricaBelfiore , why commenting it and not removing it?

Reply:
My fault, I forgot to delete the comments but the function is no more used and it will be removed